war/src/main/less/base/style.less

```diff
@@ -392,6 +392,11 @@ pre.console {
   font-size: 12pt;
 }
 
+.jenkins-not-applicable {
```

Comment:
I'm curious: why the change to the less? Something functionality related or you just considered it's a better place?

Reply:
It didn't seem to work without moving it. Possibly to do with the dev-mode setup locally but it wasn't getting loaded